src/catchup/CatchupWork.cpp

```diff
@@ -342,10 +342,12 @@ CatchupWork::getAndMaybeSetHistoryArchiveState()
                     ? CatchupConfiguration::CURRENT
                     : mApp.getHistoryManager().checkpointContainingLedger(
                           mCatchupConfiguration.toLedger());
+
```

Comment:
Nit: Extra space. Everyone has a bit of a different newline style, but we like to keep the diffs clean.